build.sbt

```diff
@@ -50,7 +49,7 @@ def commonSettings(scalaV: String) =
       "nexus" at sys.env
         .getOrElse("nexus", "https://nexus.touk.pl/nexus/content/groups/public")
     ),
-    scalaVersion                     := scalaV,
+    scalaVersion                     := scala212V,
```

Comment:
what about scala 2.13?

Reply:
I've added cross compilation with 2.13, but it was painful. Changes in code were minimal, but I think its basically impossible to make it work for flink 1.14. The module with model compatibility has a dependency to flink-streaming, which for 1.14 targets scala 2.12 and 2.11. Only from Flink 1.15 flink-streaming-java is scala version independent. As far as I understand it, because of this, it can't work.

Anyways, I got rid of 1.14 compatibility modules and now it works. I don't think we should care about compatibility that far.

Reply:
Also, I've downgraded the sbt version to 1.9.7 - the same as our main repo. With the 1.10.2 when compiling with 2.13.12, the scalaVersion was set to 2.13.14 through dependencies. And I think we should have the same 2.13 version as in main repo.

The error was:
```
[error] java.lang.RuntimeException: expected `commonTest/scalaVersion` to be "2.13.14" or later,
[error] but found "2.13.12"; upgrade scalaVersion to fix the build.
[error] 
[error] to support backwards-only binary compatibility (SIP-51),
[error] the Scala 2.13 compiler cannot be older than scala-library on the
[error] dependency classpath.
[error] see `commonTest/evicted` to know why scala-library 2.13.14 is getting pulled in.
```